app/assets/javascripts/global.js

```diff
@@ -28,6 +28,15 @@ $(document).ready(function() {
     $( ".timepicker" ).datetimepicker({
       format: 'h:mm A'
     });
+
+    //Adds listeners to select inputs to toggle disclaimers on rental schedule page
+    $(document.getElementsByName("rental[item_type_id]")).each(function() {
+      $(this).change(function () {
+        $(this).closest("form").find(".disclaimer").each(function() {
+          $(this).toggleClass("disclaimer-hidden")
+        })
+      })
+    })
```

Comment:
Oh, it will be back. I just wanted to confirm before reposting it. Naw, it is cool. Keep it this way and listen for the slow clap:

![slow_clap_citizen_kane](https://cloud.githubusercontent.com/assets/129601/17439851/e3caa81e-5af7-11e6-8a0d-bf84bf9e90c5.gif)